assets/src/edit-story/components/panels/panel/panel.js

```diff
@@ -47,12 +48,15 @@ function Panel({
   canCollapse = true,
   collapsedByDefault = true,
   initialHeight = null,
+  maxHeight: _maxHeight = MAX_HEIGHT_DEFAULT,
   ariaLabel = null,
   ariaHidden = false,
   isPersistable = true,
   isToggleDisabled,
   ...rest
 }) {
+  // If max height is 0 - fallback to default
+  const maxHeight = _maxHeight === 0 ? MAX_HEIGHT_DEFAULT : _maxHeight;
```

Comment:
SO this may be confusing - here's why this is here:

If the `maxHeight` is 0 then the layers panel will start collapsed. There was two ways to fix this:
1. Don't pass in `maxHeight` until we know the number of layers
2. Ignore `maxHeight` If it's 0.

I went with option 2 because it felt like `maxHeight` should never be zero, so it's not a use case we should respect.